src/pages/workspaces/workspace/notebooks/NotebookLauncher.js

```diff
@@ -32,37 +91,300 @@ const NotebookLauncher = _.flow(
+        onClick: () => onCopy()
+      }, ['MAKE A COPY']),
+      h(ButtonPrimary, {
+        onClick: () => onPlayground()
+      }, ['RUN IN PLAYGROUND MODE'])
+    ])
+  ])
+}
+
+const EditModeDisabledModal = ({ onDismiss, onRecreateCluster, onPlayground }) => {
+  return h(Modal, {
+    width: 700,
+    title: 'Cannot Edit Notebook',
+    onDismiss,
+    showButtons: false
+  }, [
+    p('We’ve released important updates that are not compatible with the older runtime associated with this workspace. To enable Edit Mode, please delete your existing runtime and create a new runtime.'),
+    p('If you have any files on your old runtime that you want to keep, you can access your old runtime using the Playground Mode option.'),
+    h(Link, {
+      href: 'https://support.terra.bio/hc/en-us/articles/360026639112',
+      ...Utils.newTabLinkProps
+    }, ['Read here for more details.']),
+    div({ style: { marginTop: '2rem' } }, [
+      h(ButtonSecondary, {
+        style: { paddingRight: '1rem', paddingLeft: '1rem' },
+        onClick: () => onDismiss()
+      }, ['CANCEL']),
+      h(ButtonSecondary, {
+        style: { paddingRight: '1rem', paddingLeft: '1rem', marginLeft: '1rem' },
+        onClick: () => onPlayground()
+      }, ['RUN IN PLAYGROUND MODE']),
+      h(ButtonPrimary, {
+        style: { paddingRight: '1rem', paddingLeft: '1rem', marginLeft: '2rem' },
+        onClick: () => onRecreateCluster()
+      }, ['RECREATE NOTEBOOK RUNTIME'])
+    ])
+  ])
+}
+
+const PlaygroundModal = ({ onDismiss, onPlayground }) => {
+  const [hidePlaygroundMessage, setHidePlaygroundMessage] = useState(false)
+  return h(Modal, {
+    width: 530,
+    title: 'Playground Mode',
+    onDismiss,
+    okButton: h(ButtonPrimary,
+      {
+        onClick: () => {
+          BrowserStorage.setLocalPref('hidePlaygroundMessage', hidePlaygroundMessage)
+          onPlayground()
+        }
+      },
+      'Continue')
+  }, [
+    p(['Playground mode allows you to explore, change, and run the code, but your edits will not be saved.']),
+    p(['To save your work, choose ', span({ style: { fontWeight: 600 } }, ['Download ']), 'from the ', span({ style: { fontWeight: 600 } }, ['File ']), 'menu.']),
+    h(LabeledCheckbox, {
+      checked: hidePlaygroundMessage,
+      onChange: v => setHidePlaygroundMessage(v)
+    }, [span({ style: { marginLeft: '0.5rem' } }, ['Do not show again '])])
+  ])
+}
+
+const PlaygroundHeader = () => {
+  return div({ style: { backgroundColor: colors.warning(0.7), display: 'flex', alignItems: 'center', borderBottom: `2px solid ${colors.dark(0.2)}`, height: '3.5rem', whiteSpace: 'pre' } }, [
+    div({ style: { fontSize: 18, fontWeight: 'bold', backgroundColor: colors.warning(0.9), paddingRight: '4rem', paddingLeft: '4rem', height: '100%', display: 'flex', alignItems: 'center' } },
+      ['PLAYGROUND MODE']),
+    span({ style: { marginLeft: '2rem' } }, ['Edits to this notebook are ']),
```

Comment:
The next several text/link items should be inside this span, which you might then make a div.